README.md

```diff
@@ -20,6 +20,8 @@ level of backwards compatibility to the official releases.
 
 ## Table of contents
 
+- [Releases and Versioning](#releases-and-versioning)
+  - [Currently Maintained Versions](#currently-maintained-versions)
```

Comment:
If "releases and versioning" wasn't the first point, then I'd like this one to be the first one, since it's pretty important for everybody but also easy to read.

Reply:
Yes, it just lacks context without the Releases and Versioning paragraph, but open to modifications/suggestions. :)